packages/hash/frontend/src/components/layout/PageHeader/AccountDropdown.tsx

```diff
@@ -1,51 +1,168 @@
-import { Menu } from "@headlessui/react";
-import { VoidFunctionComponent } from "react";
+import { useRef, useState, VoidFunctionComponent } from "react";
 import { tw } from "twind";
+import {
+  Box,
+  Typography,
+  Divider,
+  ListItemButton,
+  Button,
+  useTheme,
+} from "@mui/material";
 
-import { AvatarIcon, DropdownIcon } from "../../icons";
+import { UserFieldsFragment } from "../../../graphql/apiTypes.gen";
+import { Popover } from "../../Popover";
+import { Link } from "../../Link";
 
 type AccountDropdownProps = {
-  name?: string;
   avatar?: string;
   logout: () => void;
+  user: UserFieldsFragment;
 };
 
 export const AccountDropdown: VoidFunctionComponent<AccountDropdownProps> = ({
-  name,
   avatar,
   logout,
-}) => (
-  <Menu as="div" className={tw`relative`}>
-    <Menu.Button
-      title={name}
-      className="flex items-center relative z-10 m-auto py-1 px-4 focus:outline-none"
-    >
-      {avatar ? (
-        <img
-          alt="avatar"
-          src={avatar}
-          className={tw`h-6 w-6 border border(solid gray-200) rounded-full mr-3`}
-        />
-      ) : (
-        <AvatarIcon
-          className={tw`border border(solid gray-200) rounded-full mr-3`}
-        />
-      )}
-      <span className={tw`mr-2 font-bold`}>Account</span>
-      <DropdownIcon />
-    </Menu.Button>
-    <Menu.Items
-      className={tw`absolute left-0 top-0 z-0 w-full px-4 pt-10 pb-2 bg-white border-1 rounded-md flex flex-col items-end text-right`}
-    >
-      <Menu.Item>
-        <button
-          type="button"
-          onClick={logout}
-          className={tw`text-sm font-light border(b-1 transparent hover:gray-200) `}
-        >
-          Sign Out
-        </button>
-      </Menu.Item>
-    </Menu.Items>
-  </Menu>
-);
+  user,
+}) => {
+  const theme = useTheme();
+
+  const buttonRef = useRef(null);
+
+  const [open, setOpen] = useState(false);
+
+  const id = open ? "account-popover" : undefined;
+
+  return (
+    <Box>
+      <Button
```

Comment:
This is missing a the following hover tooltip:
<img width="204" alt="image" src="https://user-images.githubusercontent.com/42802102/156575910-54ecff8e-3611-4169-af59-a3bef8ee4af7.png">
If you want to do this in a follow-up, please create a task.

Reply:
Tooltips have been added now: https://github.com/hashintel/hash/commit/0ec1a4dacba243869be6daa2cd58f6c371a50a01